packages/components/app/styles/components/pagination-bar.scss

```diff
@@ -53,7 +53,10 @@
     $left: 5px,
   );
   color: inherit;
+  font: inherit;
```

Comment:
I haven't looked at the CSS yet. Let me know if want me to start to have a look.

Reply:
Thanks, I'm going through your other comments but the CSS is not yet ready (I need to work on layout differences for different combinations of sub-components after this).